qkeras/quantizers.py

```diff
@@ -574,7 +1129,7 @@ def __call__(self, x):
       # We only deal with the symmetric case right now.
       assert self.symmetric, "Only symmetric quantizers are implemented"
       len_axis = len(x.shape)
-      if len_axis > 1:
+      if len_axis != 1:
```

Comment:
I think this change is out of scope for this PR

Reply:
Addressed